package.json

```diff
@@ -65,11 +72,14 @@
     "diff": "3.5.0",
     "doctrine": "2.1.0",
     "dom-helpers": "3.3.1",
+    "dom-react": "2.2.1",
```

Comment:
Yeah, this isn't ideal. This shouldn't be an issue as soon as we move all other folders to packages. We are on it. `blocks` and `components` folders are next.